### ISSUE TITLE
Refactor primary/source/destination migrator configuration

### ISSUE THREAD

**Issue by jewzaam** [lightblue-platform/lightblue-puppet#125]
It's duplicated code.  Would be good to have a define that is reused.  Logic for setting up values would remain where it is now.

**Comment by dcrissman** [comment]
I agree. It does seem like there should be a way to condense and abstract this repeating functionality.
